calico-vpp-agent/connectivity/ipsec.go

```diff
@@ -392,19 +404,18 @@ func (p *IpsecProvider) DelConnectivity(cn *common.NodeConnectivity) (err error)
 	}
 	err = p.vpp.RouteDel(routeToDelete)
 	if err != nil {
-		return errors.Wrapf(err, "Error deleting route ipip tunnel %v: %v", tunnels)
+		p.log.Errorf("Error deleting route ipip tunnel %v: %v", tunnels, err)
```

Comment:
Is it not an issue in vpp to remove the tunnel if there is still a route using it?

Reply:
It's debatable, I'd say it's better to still try to remove the tunnel if needed. 
But whatever we do, we might have leftover state at this point.
It's just that in case of a fail due to a missing route it's better to try to run the rest of the delete